webapp/crashstats/documentation/jinja2/docs/supersearch/api.html

```diff
@@ -146,7 +146,7 @@ <h2 id="param-_aggs.*">_aggs.*</h2>
           run an aggregation on <code>field_1</code>, then for each bucket of
           that aggregation, it will aggregate on <code>field_2</code>, and then
           for each bucket of that sub-aggregation, it will aggregate on
-          <code>field_3</code>. Theoratically, we could have any number of
```

Comment:
_Theoradically_ should be a word. It's not misspelled that way here, but it made me think a bit about it.